app/src/main/java/com/github/se/eventradar/model/repository/event/MockEventRepository.kt

```diff
@@ -58,7 +61,8 @@ class MockEventRepository : IEventRepository {
   override suspend fun getEventsByIds(ids: List<String>): Resource<List<Event>> {
     val events = mutableListOf<Event>()
     for (id in ids) {
-      val event = mockEvents.find { it.fireBaseID == id }
+      val validEvents = mockEvents.filter { it.end.isAfter(LocalDateTime.now()) } // added
```

Comment:
`getEventsByIds(ids: List<String>)` should not filter to maintain consistency with the `FirebaseEventRepository`.